<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>List of Metrics • PTXQC</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="List of Metrics">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">PTXQC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.3</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/PTXQC-Basic_Guide_for_R_users.html">Basic R-Usage Guide for PTXQC</a>
    </li>
    <li>
      <a href="../articles/PTXQC-CustomizeReport.html">Customizing PTXQC reports</a>
    </li>
    <li>
      <a href="../articles/PTXQC-DragNDrop.html">Drag'n'drop Usage Guide for PTXQC</a>
    </li>
    <li>
      <a href="../articles/PTXQC-FAQ.html">Frequently asked questions for PTXQC</a>
    </li>
    <li>
      <a href="../articles/PTXQC-Input_And_Output_Data.html">Input/Output data for PTXQC</a>
    </li>
    <li>
      <a href="../articles/PTXQC-ListOfMetrics.html">List of Metrics</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/cbielow/PTXQC/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>List of Metrics</h1>
                        <h4 data-toc-skip class="author">Chris Bielow <a href="mailto:chris.bielow@fu-berlin.de" class="email">chris.bielow@fu-berlin.de</a>
</h4>
            
            <h4 data-toc-skip class="date">2025-07-15</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/cbielow/PTXQC/blob/HEAD/vignettes/PTXQC-ListOfMetrics.Rmd" class="external-link"><code>vignettes/PTXQC-ListOfMetrics.Rmd</code></a></small>
      <div class="hidden name"><code>PTXQC-ListOfMetrics.Rmd</code></div>

    </div>

    
    
<!-- the TOC (table of content) would go here, but we create it manually to use this as a vignette
      - - > use 'toc: no' above to disable the TOC 
 -->
<p><a id="TOP"></a></p>
<style type="text/css">
  p { 
    padding: 5px 0;
  }

  .helpText {
    clear: left;
  }
  
</style>
<div class="section level2">
<h2 id="table-of-contents">Table of Contents<a class="anchor" aria-label="anchor" href="#table-of-contents"></a>
</h2>
<ul>
<li><a href="#Overview">Overview</a></li>
<li>
<a href="#Metrics">Metrics</a>
<ul>
<li><a href="#AverageOverallQuality">Average Overall Quality</a></li>
<li><a href="#EVDCarryOver">EVD: CarryOver</a></li>
<li><a href="#EVDCharge">EVD: Charge</a></li>
<li><a href="#EVDIDrateoverRT">EVD: ID rate over RT</a></li>
<li><a href="#EVDMBRAlign">EVD: MBR Align</a></li>
<li><a href="#EVDMBRauxilliary">EVD: MBR auxilliary</a></li>
<li><a href="#EVDMBRIDTransfer">EVD: MBR ID-Transfer</a></li>
<li><a href="#EVDPepMissingValues">EVD: Pep Missing Values</a></li>
<li><a href="#EVDPeptideVarMod">EVD: Peptide VarMod</a></li>
<li><a href="#EVDMS2Oversampling">EVD: MS2 Oversampling</a></li>
<li><a href="#EVDPeptideCount">EVD: Peptide Count</a></li>
<li><a href="#EVDPeptideIntensity">EVD: Peptide Intensity</a></li>
<li><a href="#EVDMSCalPost">EVD: MS Cal-Post</a></li>
<li><a href="#EVDMSCalPre">EVD: MS Cal-Pre</a></li>
<li><a href="#EVDProteinCount">EVD: Protein Count</a></li>
<li><a href="#EVDReporterintensity">EVD: Reporter intensity</a></li>
<li><a href="#EVDRTPeakWidth">EVD: RT Peak Width</a></li>
<li><a href="#EVDContaminants">EVD: Contaminants</a></li>
<li><a href="#EVDUpSet">EVD: UpSet</a></li>
<li><a href="#EVDUserContaminant">EVD:User Contaminant</a></li>
<li><a href="#MSMSMissedCleavagess">MSMS: Missed Cleavages s</a></li>
<li><a href="#MSMSMS2Cal">MSMS: MS2 Cal</a></li>
<li><a href="#MS2ScansDependentPeps">MS2 Scans: Dependent Peps</a></li>
<li><a href="#MS2ScansIonInjTime">MS2 Scans: Ion Inj Time</a></li>
<li><a href="#MS2ScansIntensity">MS2 Scans: Intensity</a></li>
<li><a href="#MS2ScansTopNhigh">MS2 Scans: TopN high</a></li>
<li><a href="#MS2ScansTopNIDoverN">MS2 Scans: TopN ID over N</a></li>
<li><a href="#MS2ScansTopNoverRT">MS2 Scans: TopN over RT</a></li>
<li><a href="#PARMQParameters">PAR: MQ Parameters</a></li>
<li><a href="#PGContaminants">PG: Contaminants</a></li>
<li><a href="#PGLFQintensity">PG: LFQ intensity</a></li>
<li><a href="#PGPrincipalComponent">PG: Principal Component</a></li>
<li><a href="#PGRatio">PG: Ratio</a></li>
<li><a href="#PGRawintensity">PG: Raw intensity</a></li>
<li><a href="#PGReporterintensity">PG: Reporter intensity</a></li>
<li><a href="#SMMS2IDrate">SM: MS2 ID rate</a></li>
<li><a href="#SMTIC">SM: TIC</a></li>
</ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="overview">Overview <a name="Overview"></a><a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p>The following metrics are implemented in PTXQC.<br></p>
<p>Reasons why metrics might not appear in every report</p>
<ul>
<li>applicable only to certain types of data, e.g. SILAC or TMT</li>
<li>metric was disabled manually using the YAML config file</li>
<li>missing input data (incomplete tables), i.e. some .txt files are
missing</li>
</ul>
<p>Metric names are prefixed by the data-source, e.g. “EVD: Charge”
displays and scores the charge distribution, based on data from
‘evidence.txt’ (EVD).</p>
<ul>
<li>EVD: evidence.txt</li>
<li>SM: summary.txt</li>
<li>PAR: parameters.txt</li>
<li>PG: proteingroups.txt</li>
<li>MSMS: msms.txt</li>
<li>MS2Scans: msmsscans.txt</li>
</ul>
<p>Some metrics are incompletely named here, since they might contain
(hithero unknown) thresholds, which are taken from mqpar.xml (when
available) or defaulted in real reports.</p>
</div>
<div class="section level2">
<h2 id="metrics">Metrics <a name="Metrics"></a><a class="anchor" aria-label="anchor" href="#metrics"></a>
</h2>
<div class="section level3">
<h3 id="average-overall-quality">Average Overall Quality <a name="AverageOverallQuality"></a><a class="anchor" aria-label="anchor" href="#average-overall-quality"></a>
</h3>
<div class="helpText">
Internal metric to compute the average quality across all other metrics
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-carryover">EVD: CarryOver <a name="EVDCarryOver"></a><a class="anchor" aria-label="anchor" href="#evd-carryover"></a>
</h3>
<div class="helpText">
<p>Sample carryover … not ready yet…</p>
<p>Heatmap score [EVD: Carryover]: The percentage of peptide
identifications whose sequence gives rise to a large ‘retention
span’.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-charge">EVD: Charge <a name="EVDCharge"></a><a class="anchor" aria-label="anchor" href="#evd-charge"></a>
</h3>
<div class="helpText">
<p>Charge distribution per Raw file. For typtic digests, peptides of
charge 2 (one N-terminal and one at tryptic C-terminal R or K residue)
should be dominant. Ionization issues (voltage?), in-source
fragmentation, missed cleavages and buffer irregularities can cause a
shift (see Bittremieux 2017, DOI: 10.1002/mas.21544). The charge
distribution should be similar across Raw files. Consistent charge
distribution is paramount for comparable 3D-peak intensities across
samples.</p>
<p>Heatmap score [EVD: Charge]: Deviation of the charge 2 proportion
from a representative Raw file (‘qualMedianDist’ function).</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-id-rate-over-rt">EVD: ID rate over RT <a name="EVDIDrateoverRT"></a><a class="anchor" aria-label="anchor" href="#evd-id-rate-over-rt"></a>
</h3>
<div class="helpText">
<p>Judge column occupancy over retention time. Ideally, the LC gradient
is chosen such that the number of identifications (here, after FDR
filtering) is uniform over time, to ensure consistent instrument duty
cycles. Sharp peaks and uneven distribution of identifications over time
indicate potential for LC gradient optimization. See <a href="https://pubmed.ncbi.nlm.nih.gov/24700534/" class="external-link">Moruz 2014, DOI:
10.1002/pmic.201400036</a> for details.</p>
<p>Heatmap score [EVD: ID rate over RT]: Scored using ‘Uniform’ scoring
function, i.e. constant receives good score, extreme shapes are bad.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-mbr-align">EVD: MBR Align <a name="EVDMBRAlign"></a><a class="anchor" aria-label="anchor" href="#evd-mbr-align"></a>
</h3>
<div class="helpText">
<p>MBR Alignment: First of two steps (1=align, 2=transfer) during
Match-between-runs. This plot is based purely on real MS/MS ids.
Ideally, RTs of identical peptides should be equal (i.e. very small
residual RT delta) across Raw files after alignment.</p>
<p>MaxQuants RT correction is shown in blue – it should be well within
the alignment search window (20min by default) set during MaxQuant
configuration. The resulting residual RT delta after RT alignment
(compared to a reference Raw file), is shown as green/red dots. One dot
represents one peptide (incl. charge). Every dot (peptide) outside an
allowed residual delta RT (1min by default) is colored red. All others
are green. The ratio of ‘green’ vs. ‘green+red’ peptides is annotated
using ‘sc:’ (for ‘score’) in the plot subtitles. High values are better
(green peptides dominate).</p>
<p>If moving ‘red’ dots to the horizontal zero-line (to make them green)
requires large RT shifts, then increasing the alignment search window
might help MaxQuant to find a better alignment.</p>
<p>Heatmap score [EVD: MBR Align]: ratio of ‘green’ vs. ‘green+red’
peptides</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-mbr-auxilliary">EVD: MBR auxilliary <a name="EVDMBRauxilliary"></a><a class="anchor" aria-label="anchor" href="#evd-mbr-auxilliary"></a>
</h3>
<div class="helpText">
<p>Auxililiary plots – experimental – without scores.</p>
<p>Return a tree plot with a possible alignment tree. This allows the
user to judge which Raw files have similar corrected RT’s (i.e. where
aligned successfully). If there are clear sub-clusters, it might be
worth introducing artifical fractions into MaxQuant, to avoid
ID-transfer between these clusters (use the MBR-Align and
MBR-ID-Transfer metrics to support the decision).</p>
<p>If the input contains fractions, leaf nodes will be colored
accordingly. Distinct sub-clusters should have their own color. If not,
MaxQuant’s fraction settings should be optimized. Note that introducing
fractions in MaxQuant will naturally lead to a clustering here (it’s
somewhat circular).</p>
<p>Heatmap score: none.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-mbr-id-transfer">EVD: MBR ID-Transfer <a name="EVDMBRIDTransfer"></a><a class="anchor" aria-label="anchor" href="#evd-mbr-id-transfer"></a>
</h3>
<div class="helpText">
<p>MBR Transfer: Last of two steps (1=align, 2=transfer) during
Match-between-runs. If MaxQuant only transfers peptide ID’s which are
not present in the target file, then each Raw file should not have any
duplicates of identical peptides (incl. charge). Sometimes, a single or
split 3D-peak gets annotated multiple times, that’s ok. However, the
same peptide should not be annotated twice (or more) at vastly different
points in RT.</p>
<p>This plot shows three columns: - left: the ‘genuine’ situation
(pretending that no MBR was computed) - middle: looking only at
transferred IDs - right: combined picture (a mixture of left+middle,
usually)</p>
<p>Each peptide falls into three categories (the colors): - single
(good, because it has either one genuine OR a transferred ID). -
in-group (also good, because all ID’s are very close in RT) - out-group
(bad, spread across the RT gradient – should not be possible; a false
ID)</p>
<p>Heatmap score [EVD: MBR ID-Transfer]: The fraction of non-out-group
peptides (i.e. good peptides) in the middle column. This score is
‘pessimistic’ because if few ID’s were transferred, but all of them are
bad, the score is bad, even though the majority of peptides is still ok
(because they are genuine). However, in this case MBR provides few (and
wrong) additional information, and should be disabled.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-pep-missing-values">EVD: Pep Missing Values <a name="EVDPepMissingValues"></a><a class="anchor" aria-label="anchor" href="#evd-pep-missing-values"></a>
</h3>
<div class="helpText">
<p>Missing peptide intensities per Raw file from evidence.txt. This
metric shows the fraction of missing peptides compared to all peptides
seen in the whole experiment. The more Raw files you have, the higher
this fraction is going to be (because there is always going to be some
exotic [low intensity?] peptide which gets [falsely] identified in only
a single Raw file). A second plot shows how many peptides (Y-axis) are
covered by at least X Raw files. A third plot shows the density of the
observed (line) and the missing (filled area) data. To reconstruct the
distribution of missing values, an imputation strategy is required, so
the argument is somewhat circular here. If all Raw files are (technical)
replicates, i.e. we can expect that missing peptides are indeed present
and have an intensity similar to the peptides we do see, then the median
is a good estimator. This method performs a global normalization across
Raw files (so their observed intensitiy distributions have the same
mean), before computing the imputed values. Afterwards, the
distributions are de-normalized again (shifting them back to their)
original locations – but this time with imputed peptides.</p>
<p>Peptides obtained via Match-between-run (MBR) are accounted for
(i.e. are considered as present = non-missing). Thus, make sure that MBR
is working as intended (see MBR metrics).</p>
<p><b>Warning:</b> this metric is meaningless for fractionated data!
<b>TODO:</b> compensate for lower scores in large studies (with many Raw
files), since peptide FDR is accumulating!?</p>
<p>Heatmap score [EVD: Pep Missing]: Linear scale of the fraction of
missing peptides.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-peptide-varmod">EVD: Peptide VarMod <a name="EVDPeptideVarMod"></a><a class="anchor" aria-label="anchor" href="#evd-peptide-varmod"></a>
</h3>
<div class="helpText">
<p>Compute an occurence table of modifications (e.g. Oxidation (M)) for
all peptides, including the unmodified.</p>
<p>The plot will show percentages, i.e. is normalized by the total
number of peptide sequences (where different charge state counts as a
separate peptide) per Raw file.</p>
<p>The sum of frequencies may exceed 100% per Raw file, since a peptide
can have multiple modifications. E.g. given three peptides in a single
Raw file <br> 1. <em>M(Oxidation (M))LVLDEADEM(Oxidation (M))LNK</em>
<br> 2. <em>(Acetyl (Protein N-term))M(Oxidation (M))YGLLLENLSEYIK</em>
<br> 3. DPFIANGER <br></p>
<p>, the following frequencies arise:</p>
<ul>
<li>33% of ‘Acetyl (Protein N-term)’ <br>
</li>
<li>33% of ‘Oxidation (M)’ <br>
</li>
<li>33% of ‘2 Oxidation (M)’ <br>
</li>
<li>33% of ‘Unmodified’ <br>
</li>
</ul>
<p>Thus, 33% of sequences are unmodified, implying 66% are modified at
least once. If a modification, e.g. Oxidation(M), occurs multiple times
in a single peptide it’s listed as a separate modification (here ‘2
Oxidation (M)’).</p>
<p>Heatmap score [EVD: Pep ModTable]: Deviation of (unmodified peptides
fraction) when compared to a representative Raw file (‘qualMedianDist’
function).</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-ms2-oversampling">EVD: MS2 Oversampling <a name="EVDMS2Oversampling"></a><a class="anchor" aria-label="anchor" href="#evd-ms2-oversampling"></a>
</h3>
<div class="helpText">
<p>An oversampled 3D-peak is defined as a peak whose peptide ion (same
sequence and same charge state) was identified by at least two distinct
MS<sup>2</sup> spectra in the same Raw file. For high complexity
samples, oversampling of individual 3D-peaks automatically leads to
undersampling or even omission of other 3D-peaks, reducing the number of
identified peptides. Oversampling occurs in low-complexity samples or
long LC gradients, as well as undersized dynamic exclusion windows for
data independent acquisitions.</p>
<pre><code>If DIA-Data: this metric is skipped</code></pre>
<p>Heatmap score [EVD: MS<sup>2</sup> Oversampling]: The percentage of
non-oversampled 3D-peaks.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-peptide-count">EVD: Peptide Count <a name="EVDPeptideCount"></a><a class="anchor" aria-label="anchor" href="#evd-peptide-count"></a>
</h3>
<div class="helpText">
<p>Number of unique (i.e. not counted twice) peptide sequences including
modifications (after FDR) per Raw file. A configurable target threshold
is indicated as dashed line.</p>
<p>If MBR was enabled, three categories (‘genuine (exclusive)’, ‘genuine
+ transferred’, ‘transferred (exclusive)’ are shown, so the user can
judge the gain that MBR provides. Peptides in the ‘genuine +
transferred’ category were identified within the Raw file by MS/MS, but
at the same time also transferred to this Raw file using MBR. This ID
transfer can be correct (e.g. in case of different charge states), or
incorrect – see MBR-related metrics to tell the difference. Ideally, the
‘genuine + transferred’ category should be rather small, the other two
should be large.</p>
<p>If MBR would be switched off, you can expect to see the number of
peptides corresponding to ‘genuine (exclusive)’ + ‘genuine +
transferred’. In general, if the MBR gain is low and the MBR scores are
bad (see the two MBR-related metrics), MBR should be switched off for
the Raw files which are affected (could be a few or all).</p>
<p>Heatmap score [EVD: Pep Count (&gt;%1.0f)]: Linear scoring from zero.
Reaching or exceeding the target threshold gives a score of 100%%.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-peptide-intensity">EVD: Peptide Intensity <a name="EVDPeptideIntensity"></a><a class="anchor" aria-label="anchor" href="#evd-peptide-intensity"></a>
</h3>
<div class="helpText">
<p>Peptide precursor intensity per Raw file from evidence.txt WITHOUT
match-between-runs evidence. Low peptide intensity usually goes hand in
hand with low MS/MS identifcation rates and unfavourable signal/noise
ratios, which makes signal detection harder. Also instrument acquisition
time increases for trapping instruments.</p>
<p>Failing to reach the intensity threshold is usually due to
unfavorable column conditions, inadequate column loading or ionization
issues. If the study is not a dilution series or pulsed SILAC
experiment, we would expect every condition to have about the same
median log-intensity (of 2<sup>%1.1f</sup>). The relative standard
deviation (RSD) gives an indication about reproducibility across files
and should be below 5%%.</p>
<p>Depending on your setup, your target thresholds might vary from
PTXQC’s defaults. Change the threshold using the YAML configuration
file.</p>
<p>Heatmap score [EVD: Pep Intensity (&gt;%1.1f)]: Linear scale of the
median intensity reaching the threshold, i.e. reaching 2<sup>21</sup> of
2<sup>23</sup> gives score 0.25.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-ms-cal-post">EVD: MS Cal-Post <a name="EVDMSCalPost"></a><a class="anchor" aria-label="anchor" href="#evd-ms-cal-post"></a>
</h3>
<div class="helpText">
<p>Precursor mass accuracy after calibration. Failed samples from
precalibration data are still marked here. Ppm errors should be centered
on zero and their spread is expected to be significantly smaller than
before calibration.</p>
<p>Heatmap score [EVD: MS Cal-Post]: The variance and centeredness
around zero of the calibrated distribution (function GaussDev).</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-ms-cal-pre">EVD: MS Cal-Pre <a name="EVDMSCalPre"></a><a class="anchor" aria-label="anchor" href="#evd-ms-cal-pre"></a>
</h3>
<div class="helpText">
<p>Mass accurary before calibration. Outliers are marked as such
(‘out-of-search-tol’) using ID rate and standard deviation as additional
information (if available). If any Raw file is flagged ‘failed’,
increasing MaxQuant’s first-search tolerance (20ppm by default, here:
%1.1f ppm) might help to enable successful recalibration. A bug in
MaxQuant sometimes leads to excessively high ppm mass errors
(&gt;10<sup>4</sup>) reported in the output data. However, this can
sometimes be corrected for by re-computing the delta mass error from
other data. If this is the case, a warning (‘bugfix applied’) will be
shown.</p>
<p>Heatmap score [EVD: MS Cal Pre (%1.1f)]: the centeredness (function
CenteredRef) of uncalibrated masses in relation to the search window
size.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-protein-count">EVD: Protein Count <a name="EVDProteinCount"></a><a class="anchor" aria-label="anchor" href="#evd-protein-count"></a>
</h3>
<div class="helpText">
<p>Number of Protein groups (after FDR) per Raw file. A configurable
target threshold is indicated as dashed line.</p>
<p>If MBR was enabled, three categories (‘genuine (exclusive)’, ‘genuine
+ transferred’, ‘transferred (exclusive)’ are shown, so the user can
judge the gain that MBR provides. Here, ‘transferred (exclusive)’ means
that this protein group has peptide evidence which originates only from
transferred peptide IDs. The quantification is (of course) always from
the local Raw file. Proteins in the ‘genuine + transferred’ category
have peptide evidence from within the Raw file by MS/MS, but at the same
time also peptide IDs transferred to this Raw file using MBR were used.
It is not unusual to see the ‘genuine + transferred’ category be the
rather large, since a protein group usually has peptide evidence from
both sources. To see of MBR worked, it is better to look at the two
MBR-related metrics.</p>
<p>If MBR would be switched off, you can expect to see the number of
protein groups corresponding to ‘genuine (exclusive)’ + ‘genuine +
transferred’. In general, if the MBR gain is low and the MBR scores are
bad (see the two MBR-related metrics), MBR should be switched off for
the Raw files which are affected (could be a few or all).</p>
<p>Heatmap score [EVD: Prot Count (&gt;%1.0f)]: Linear scoring from
zero. Reaching or exceeding the target threshold gives a score of
100%%.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-reporter-intensity">EVD: Reporter intensity <a name="EVDReporterintensity"></a><a class="anchor" aria-label="anchor" href="#evd-reporter-intensity"></a>
</h3>
<div class="helpText">
<p>ITRAQ/TMT reporter intensity violin plots of all PSMs for each
channel and Raw file. The second subplot shows labeling efficiency (LE),
i.e the fraction of PSMs with non-zero abundance (100% = full labeling
of all PSMs; 0% = no reporter ions at all). This is used for heatmap
scoring. See below.</p>
<p>There is a similar ‘Experimental Group’ based metric/plot based on
proteins.txt.</p>
<p>PTXQC uses isotope-corrected intensities (eliminating channel
carry-over) to allow for detection of empty channels, e.g. due to
mis-labeling. If MaxQuant did no isotope correction (i.e. corrected and
uncorrected channels are equal), the plot title will show a warning. The
scores are too optimistic in this case (since carry-over will be
mistaken for actual signal).</p>
<p>Note: global labelling efficiency can only be judged indirectly with
this metric, since isobaric reporters where set as fixed modification.
Thus, MaxQuant. will only identify labeled peptides in the first place.
Observing only very few peptides (see peptide count metric), is a good
indicator. However, if only the labeling of a few channels failed, this
will be noticable here!</p>
<p>Labeling can still be poor, even though identification was
successful.</p>
<p>A labeling efficiency (LE) is computed per Raw file AND channel as:
the percentage of PSMs which have non-zero reporter intensity. Ideally
LE reaches 100 percent (all peptides have an intensity in the channel;
biological missingness ignored).</p>
<p>Heatmap score: minimum labeling efficiency per Raw file across all
channels. I.e. for 4-plex ITRAQ and two Raw files, there will be 8
labeling efficiency (LE) values. Each Raw file is now scored by the
minimum LE of all its 4 channels.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-rt-peak-width">EVD: RT Peak Width <a name="EVDRTPeakWidth"></a><a class="anchor" aria-label="anchor" href="#evd-rt-peak-width"></a>
</h3>
<div class="helpText">
<p>One parameter of optimal and reproducible chromatographic separation
is the distribution of widths of peptide elution peaks, derived from the
evidence table. Ideally, all Raw files show a similar distribution,
e.g. to allow for equal conditions during dynamic precursor exclusion,
RT alignment or peptide quantification.</p>
<p>Heatmap score [EVD: RT Peak Width]: Scored using BestKS function,
i.e. the D statistic of a Kolmogoriv-Smirnoff test.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-contaminants">EVD: Contaminants <a name="EVDContaminants"></a><a class="anchor" aria-label="anchor" href="#evd-contaminants"></a>
</h3>
<div class="helpText">
<p>PTXQC will explicitly show the five most abundant external protein
contaminants (as detected via MaxQuant’s contaminants FASTA file) by Raw
file, and summarize the remaining contaminants as ‘other’. This allows
to track down which proteins exactly contaminate your sample. Low
contamination is obviously better. The ‘Abundance class’ models the
average peptide intensity in each Raw file and is visualized using
varying degrees of transparency. It is not unusual to see samples with
low sample content to have higher contamination. If you see only one
abundance class (‘mid’), this means all your Raw files have roughly the
same peptide intensity distribution.</p>
<p>If you see less than 5 contaminants, it either means there are
actually less, or that one (or more) of the shortened contaminant names
subsume multiple of the top5 contaminants (since they start with the
same prefix).</p>
<p>Heatmap score [EVD: Contaminants]: as fraction of summed intensity
with 0 = sample full of contaminants; 1 = no contaminants</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="evd-upset">EVD: UpSet <a name="EVDUpSet"></a><a class="anchor" aria-label="anchor" href="#evd-upset"></a>
</h3>
<div class="helpText">
<p>The metric shows an upSet plot based on the number of modified
peptide sequences per Raw file, intersected or merged with other Raw
files (see below for details).<br></p>
<p>If the number of Raw files is &gt;=6, only the <code>distinct</code>
plot is generated (the other two are skipped for performance reasons).
<a href="https://raw.githubusercontent.com/cbielow/PTXQC/master/inst/reportTemplate/modes_UpSet.png" target="_blank" rel="noopener" class="external-link"><span>See
here for an example plot showing how the set size is computed</span>
</a>.</p>
<p>Definition: An <code>active set</code> is the set of black dots in a
column of the plot – as opposed to the grey dots (you will understand
when you see it).</p>
<p>
<b>distinct:</b> shows the number of sequences that are present in ALL
active sets. For three Raw files and active sets A and B, this would
mean all sequences which occur in A and B (intersect), but not in C
(setdiff).<br><b>intersection:</b> shows the number of sequences that
occurs in all active sets (intersection).<br><b>union:</b> shows the
number of sequences that occurs in total. For two files that are all
sequences that occurs either in A or in B (union).<br></p>
<p>
Heatmap score [EVD: UpSet]: The proportion of sequences that the file
has in common with all other files.
</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
<div class="section level3">
<h3 id="evduser-contaminant">EVD:User Contaminant <a name="EVDUserContaminant"></a><a class="anchor" aria-label="anchor" href="#evduser-contaminant"></a>
</h3>
<div class="helpText">
<p>User defined contaminant plot based on peptide intensities and
counts. Usually used for Mycoplasma detection, but can be used for an
arbitrary (set of) proteins.</p>
<p>All proteins (and their peptides) which contain the search string
from the YAML file are considered contaminants. The contaminant’s search
string is searched in the full FASTA header in proteinGroups.txt. If
proteinGroups.txt is not available/found, only protein identifiers can
be considered. The search realm used is given in the plot subtitle. You
should choose the contaminant name to be distinctive. Only peptides
belonging to a single protein group are considered when computing the
fractions (contaminant vs. all), since peptides shared across multiple
groups are potentially false positives.</p>
<p>Two abundance measures are computed per Raw file:</p>
<ul>
<li>fraction of contaminant intensity (used for scoring of the
metric)</li>
<li>fraction of contaminant spectral counts (as comparison; both should
be similar)</li>
</ul>
<p>If the intensity fraction exceeds the threshold (indicated by the
dashed horizontal line) a contamination is assumed.</p>
<p>For each Raw file exceeding the threshold an additional plot giving
cumulative Andromeda peptide score distributions is shown. This allows
to decide if the contamination is true. Contaminant scores should be
equally high (or higher), i.e. to the right, compared to the sample
scores. Each graph’s subtitle is augmented with a p-value of the
Kologorov-Smirnoff test of this data (Andromeda scores of contaminant
peptides vs. sample peptides). If the p-value is high, there is no score
difference between the two peptide populations. In particular, the
contaminant peptides are not bad-scoring, random hits. These p-values
are also shown in the first figure for each Raw file. Note that the
p-value is purely based on Andromeda scores and is independent of
intensity or spectral counts.</p>
<p>Heatmap score [EVD: Contaminant <name>]: boolean score, i.e. 0%
(fail) if the intensity threshold was exceeded; otherwise 100%
(pass).</name></p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="msms-missed-cleavages-s">MSMS: Missed Cleavages s <a name="MSMSMissedCleavagess"></a><a class="anchor" aria-label="anchor" href="#msms-missed-cleavages-s"></a>
</h3>
<div class="helpText">
<p>Under optimal digestion conditions (high enzyme grade etc.), only few
missed cleavages (MC) are expected. In general, increased MC counts also
increase the number of peptide signals, thus cluttering the available
space and potentially provoking overlapping peptide signals, biasing
peptide quantification. Thus, low MC counts should be favored.
Interestingly, it has been shown recently that incorporation of peptides
with missed cleavages does not negatively influence protein
quantification (see <a href="https://doi.org/10.1021/pr500294d" class="external-link">Chiva,
C., Ortega, M., and Sabido, E. Influence of the Digestion Technique,
Protease, and Missed Cleavage Peptides in Protein Quantitation. J.
Proteome Res. 2014, 13, 3979-86</a> ). However this is true only if all
samples show the same degree of digestion. High missed cleavage values
can indicate for example, either a) failed digestion, b) a high
(post-digestion) protein contamination, or c) a sample with high amounts
of unspecifically degraded peptides which are not digested by
trypsin.</p>
<p>If MC&gt;=1 is high (&gt;20%) you should increase the missed
cleavages settings in MaxQuant and compare the number of peptides.
Usually high MC correlates with bad identification rates, since many
spectra cannot be matched to the forward database.</p>
<p>In the rare case that ‘no enzyme’ was specified in MaxQuant, neither
scores nor plots are shown.</p>
<p>Heatmap score [MSMS: MC]: the fraction (0% - 100%) of fully cleaved
peptides per Raw file</p>
Heatmap score [MSMS: MC Var]: each Raw file is scored for its deviation
(score: MedianDist) from the ‘average’ digestion state of the current
study.
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="msms-ms2-cal">MSMS: MS2 Cal <a name="MSMSMS2Cal"></a><a class="anchor" aria-label="anchor" href="#msms-ms2-cal"></a>
</h3>
<div class="helpText">
<p>MS/MS decalibration metric. If most of the fragments are within
tighter bounds, you can reduce the fragment mass tolerance to obtain
more identifications under the same FDR. On the other hand, if the
fragment mass errors are not centered on zero, a recalibration of the
instrument should be performed. If the (Gaussian-like) distribution is
cut too severely on either side by the search tolerance window in
MaxQuant, you might be able to increase the number of identifications by
allowing for a wider MS/MS search window when re-running MaxQuant.
However, the number of decoy identifications will increase as well,
potentially offsetting any gain when FDR is applied.</p>
<p>Heatmap score [MSMS: MS<sup>2</sup> Cal (Analyzer)]: rewards
centeredness around 0 ppm/Da (function Centered).</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-dependent-peps">MS2 Scans: Dependent Peps <a name="MS2ScansDependentPeps"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-dependent-peps"></a>
</h3>
<div class="helpText">
<p>Prominent dependent peptides are shown, and how they contribute to
increase identification numbers. You can use this metric to assess if
specifying another variable (or even fixed) modification makes sense to
boost your ID rate (remember that dependent peptides are only an add-on
in MaxQuant and do not count towards global ID rates or
quantification!). You can also the use the DP-modifications to compare
samples (e.g. with modified sample preparation or biological conditions
where you expect drastic changes).</p>
<p>Hits with ‘Unmodified’ are removed since they do not provide a lot of
additional information. The legend provides the top target sites (AAs)
for each modification in percent (e.g. ‘Oxidation - nterm(8) L(9) H(7)’
means that of all dependent peptides with Oxidation, 8% of there are
nterm, 9% are on L, 7% on H).</p>
<p>Heatmap score [MS<sup>2</sup> Scans: DepPep]: No score.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-ion-inj-time">MS2 Scans: Ion Inj Time <a name="MS2ScansIonInjTime"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-ion-inj-time"></a>
</h3>
<div class="helpText">
<p>Ion injection time score - should be as low as possible to allow fast
cycles. Correlated with peptide intensity. Note that this threshold
needs customization depending on the instrument used (e.g., ITMS
vs. FTMS).</p>
<p>Heatmap score [MS<sup>2</sup> Scans: Ion Inj Time]: Linear score as
fraction of MS/MS below the threshold.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-intensity">MS2 Scans: Intensity <a name="MS2ScansIntensity"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-intensity"></a>
</h3>
<div class="helpText">
<p>MS/MS identifications can be ‘bad’ for a couple of reasons. It could
be computational, i.e. ID rates are low because you specified the wrong
protein database or modifications (not our concern here). Another reason
is low/missing signals for fragment ions, e.g. due to bad
(quadrupole/optics) ion transmission (charging effects), too small
isolation windows, etc.</p>
<p>Hence, we plot the TIC and base peak intensity of all MS/MS scans
(incl. unidentified ones) per Raw file. Depending on the setup, these
intensities can vary, but telling apart good from bad samples should
never be a problem. If you only have bad samples, you need to know the
intensity a good sample would reach.</p>
<p>To automatically score this, we found that the TIC should be 10-100x
larger than the base peak, i.e. there should be many other ions which
are roughly as high (a good fragmentation ladder). If there are only a
few spurious peaks (bad MS/MS), the TIC is much lower. Thus, we score
the ratio BP * 10 &lt; TIC (this would be 100% score). If it’s only BP *
3 &gt; TIC, we say this MS/MS failed (0%). Anything between 3x and 10x
gets a score in between. The score for the Raw file is computed as the
median score across all its MS/MS scans.</p>
Heatmap score [MS<sup>2</sup> Scans: Intensity]: Linear score (0-100%)
between 3 &lt; (TIC / BP) &lt; 10.
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-topn-high">MS2 Scans: TopN high <a name="MS2ScansTopNhigh"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-topn-high"></a>
</h3>
<div class="helpText">
<p>Reaching TopN on a regular basis indicates that all sections of the
LC gradient deliver a sufficient number of peptides to keep the
instrument busy. This metric somewhat summarizes ‘TopN over RT’.</p>
<p>Heatmap score [MS<sup>2</sup> Scans: TopN high]: rewards if TopN was
reached on a regular basis (function qualHighest)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-topn-id-over-n">MS2 Scans: TopN ID over N <a name="MS2ScansTopNIDoverN"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-topn-id-over-n"></a>
</h3>
<div class="helpText">
<p>Looking at the identification rates per scan event (i.e. the MS/MS
scans after a survey scan) can give hints on how well scheduled
precursor peaks could be fragmented and identified. If performance drops
for the later MS/MS scans, then the LC peaks are probably not wide
enough to deliver enough eluent or the intensity threshold to trigger
the MS/MS event should be lowered (if LC peak is already over), or
increased (if LC peak is still to weak to collect enough ions).</p>
<p>Heatmap score [MS<sup>2</sup> Scans: TopN ID over N]: Rewards uniform
identification performance across all scan events.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="ms2-scans-topn-over-rt">MS2 Scans: TopN over RT <a name="MS2ScansTopNoverRT"></a><a class="anchor" aria-label="anchor" href="#ms2-scans-topn-over-rt"></a>
</h3>
<div class="helpText">
<p>TopN over retention time. Similar to ID over RT, this metric reflects
the complexity of the sample at any point in time. Ideally complexity
should be made roughly equal (constant) by choosing a proper
(non-linear) LC gradient. See <a href="https://pubmed.ncbi.nlm.nih.gov/24700534/" class="external-link">Moruz 2014, DOI:
10.1002/pmic.201400036</a> for details.</p>
<p>Heatmap score [MS<sup>2</sup> Scans: TopN over RT]: Rewards uniform
(function Uniform) TopN events over time.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="par-mq-parameters">PAR: MQ Parameters <a name="PARMQParameters"></a><a class="anchor" aria-label="anchor" href="#par-mq-parameters"></a>
</h3>
<div class="helpText">
MaxQuant parameters, extracted from parameters.txt (abbreviated as
‘PAR’), summarizes the settings used for the MaxQuant analysis. Key
parameters are MaxQuant version, Re-quantify, Match-between-runs and
mass search tolerances. A list of protein database files is also
provided, allowing to track database completeness and database version
information (if given in the filename).
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-contaminants">PG: Contaminants <a name="PGContaminants"></a><a class="anchor" aria-label="anchor" href="#pg-contaminants"></a>
</h3>
<div class="helpText">
<p>External protein contamination should be controlled for, therefore
MaxQuant ships with a comprehensive, yet customizable protein
contamination database, which is searched by MaxQuant by default. PTXQC
generates a contamination plot derived from the proteinGroups (PG) table
showing the fraction of total protein intensity attributable to
contaminants. The plot employs transparency to discern differences in
the group-wise summed protein abundance. This allows to delineate a high
contamination in high complexity samples from a high contamination in
low complexity samples (e.g. from in-gel digestion). If you see only one
abundance class (‘mid’), this means all your groups have roughly the
same summed protein intensity. Note that this plot is based on
experimental groups, and therefore may not correspond 1:1 to Raw
files.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-lfq-intensity">PG: LFQ intensity <a name="PGLFQintensity"></a><a class="anchor" aria-label="anchor" href="#pg-lfq-intensity"></a>
</h3>
<div class="helpText">
<p>Label-free quantification (LFQ) intensity boxplots by experimental
groups. Groups are user-defined during MaxQuant configuration. This plot
displays a (customizable) threshold line for the desired mean of LFQ
intensity of proteins. Raw files which underperform in <em>Raw</em>
intensity, are likely to show an <em>increased</em> mean here, since
only high-abundance proteins are recovered and quantifyable by MaxQuant
in this Raw file. The remaining proteins are likely to receive an LFQ
value of 0 (i.e. do not contribute to the distribution). The height of
the bar correlates to the number of proteins with non-zero
abundance.</p>
<p>Contaminants are shown as overlayed yellow boxes, whose height
corresponds to the number of contaminant proteins. The position of the
box gives the intensity distribution of the contaminants.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-principal-component">PG: Principal Component <a name="PGPrincipalComponent"></a><a class="anchor" aria-label="anchor" href="#pg-principal-component"></a>
</h3>
<div class="helpText">
<p>Principal components plots of experimental groups (as defined during
MaxQuant configuration).</p>
<p>This plot is shown only if more than one experimental group was
defined. If LFQ was activated in MaxQuant, an additional PCA plot for
LFQ intensities is shown. Similarly, if iTRAQ/TMT reporter intensities
are detected.</p>
<p>Since experimental groups and Raw files do not necessarily correspond
1:1, this plot cannot use the abbreviated Raw file names, but instead
must rely on automatic shortening of group names.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-ratio">PG: Ratio <a name="PGRatio"></a><a class="anchor" aria-label="anchor" href="#pg-ratio"></a>
</h3>
<div class="helpText">
<p>This plot shows log<sub>2</sub> ratios for SILAC-like experiments
(whenever MaxQuant reports a set of ’ratio.*’ columns). Useful to spot
unequal channel mixing during sample preparation. If equal mixing is
expected, the distribution should be unimodal and its mode close to 1
(i.e., a 1:1 ratio), as indicated by a visual guidance line. Multimodal
distributions are flagged as such automatically. If PTXQC detects ratios
deviating strongly from 1:1 (parameterized by default beyond the range
between 1:4 and 4:1), PTXQC automatically assumes a pulsed experiment
and reports the label incorporation in percent for all groups.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-raw-intensity">PG: Raw intensity <a name="PGRawintensity"></a><a class="anchor" aria-label="anchor" href="#pg-raw-intensity"></a>
</h3>
<div class="helpText">
<p>Intensity boxplots by experimental groups. Groups are user-defined
during MaxQuant configuration. This plot displays a (customizable)
threshold line for the desired mean intensity of proteins. Groups which
underperform here, are likely to also suffer from a worse MS/MS id rate
and higher contamination due to the lack of total protein
loaded/detected. If possible, all groups should show a high and
consistent amount of total protein. The height of the bar correlates to
the number of proteins with non-zero abundance.</p>
<p>Contaminants are shown as overlayed yellow boxes, whose height
corresponds to the number of contaminant proteins. The position of the
box gives the intensity distribution of the contaminants.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="pg-reporter-intensity">PG: Reporter intensity <a name="PGReporterintensity"></a><a class="anchor" aria-label="anchor" href="#pg-reporter-intensity"></a>
</h3>
<div class="helpText">
<p>ITRAQ/TMT reporter intensity boxplots by experimental groups. Groups
are user-defined during MaxQuant configuration. This plot displays a
(customizable) threshold line for the desired mean of reporter ion
intensity of proteins. The height of the bar correlates to the number of
proteins with non-zero abundance.</p>
<p>Contaminants are shown as overlayed yellow boxes, whose height
corresponds to the number of contaminant proteins. The position of the
box gives the intensity distribution of the contaminants. Contaminants
should be lower compared to label-free samples, since all contaminants
introduced after the labeling should not be identified by Andromeda
(since they lack the isobaric tag).</p>
<p>There is a similar ‘Raw file’ based metric/plot based on
evidence.txt.</p>
<p>Heatmap score: none (since data source proteinGroups.txt is not
related 1:1 to Raw files)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="sm-ms2-id-rate">SM: MS2 ID rate <a name="SMMS2IDrate"></a><a class="anchor" aria-label="anchor" href="#sm-ms2-id-rate"></a>
</h3>
<div class="helpText">
<p>MS/MS identification rate per Raw file from summary.txt (SM). Each
Raw file is colored according to its ID rate and categorized into
performance bins as ‘bad’, ‘ok’ and ‘great’. Raw files below ‘ok’, are
listed separately on the next page of the report for convenient
follow-up.</p>
<p>The thresholds for the bins are</p>
<p>%s</p>
<p>Heatmap score [SM: MS<sup>2</sup> IDrate (&gt;%1.0f)]: reaches 1
(=100%%) if the threshold for ‘great’ is reached or exceeded.</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
</div>
<div class="section level3">
<h3 id="sm-tic">SM: TIC <a name="SMTIC"></a><a class="anchor" aria-label="anchor" href="#sm-tic"></a>
</h3>
<div class="helpText">
<p>Total Ion Count: Returns the summed intensity of all MS1 signals
(regardless of identification state).</p>
<p>Heatmap score [SM: TIC]: reaches 1 (=100%%) if the TIC is uniform
(i.e. a flat line)</p>
</div>
<p style="text-align: right">
<a href="#TOP">– back to top –</a>
</p>
<p>If the above list is empty, the vignette was not compiled (e.g. when
viewing on GitHub). Please see <a href="https://CRAN.R-project.org/package=PTXQC" class="external-link uri">https://CRAN.R-project.org/package=PTXQC</a> –&gt; Vignettes
for a compiled version.</p>
</div>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Chris Bielow.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.2.</p>
</div>

      </footer>
</body>
</html>
